graylog2-web-interface/src/theme/utils/opacify.js

```diff
@@ -0,0 +1,18 @@
+import chroma from 'chroma-js';
+
+function opacify(color, amount) {
+  /**
+   * Increases the opacity of a color. Its range for the amount is between 0 to 1.
+   *
+   * @param {string} color - any string that represents a color (ex: "#f00" or "rgb(255, 0, 0)")
+   * @param {number} amount - any positive or negative number
+   */
+  if (color === 'transparent') return color;
```

Comment:
https://eslint.org/docs/rules/curly

I am confused why the linter did not see this.